examples/animation/animate.py

```diff
@@ -8,11 +8,11 @@
 '''
 
 import kivy
-kivy.require('1.0.7')
 
 from kivy.animation import Animation
 from kivy.app import App
 from kivy.uix.button import Button
+kivy.require('1.0.7')
```

Comment:
Same here, this could be probably removed too, `1.0.7` will be satisfied everywhere (unless someone feels too adventurous to visit history) :D